sphinx/source/docs/user_guide/graph.rst

```diff
@@ -44,111 +43,139 @@ Here's a code snippet that:
     from bokeh.models import GraphRenderer, Ellipse
     from bokeh.palettes import Spectral8
 
+    ### list the nodes and initialize a plot
```

Comment:
We don't use a triple hash `###` anywhere else in the docs or examples, I think it should just be `#` (with a single space after) for consistency